sdks/java/core/src/main/java/org/apache/beam/sdk/runners/inprocess/InProcessBundleFactory.java

```diff
@@ -48,9 +48,7 @@ private InProcessBundleFactory() {}
 
   @Override
   public <T> UncommittedBundle<T> createBundle(CommittedBundle<?> input, PCollection<T> output) {
-    return input.isKeyed()
-        ? InProcessBundle.keyed(output, input.getKey())
-        : InProcessBundle.unkeyed(output);
+    return InProcessBundle.keyed(output, input.getKey());
```

Comment:
`unkeyed` isn't really a thing that exists on a bundle level (after the introduction and use of [`KeyedPValueTrackingVisitor`](https://github.com/apache/incubator-beam/blob/master/sdks/java/core/src/main/java/org/apache/beam/sdk/runners/inprocess/KeyedPValueTrackingVisitor.java)) - the key has no meaning for an unkeyed `PValue`, so it can be an arbitrary value, but for all keyed `PValues` keys are explicitly assigned (as the result of a `GroupByKey`), so the value of key is meaningful and explicit.